src/zimdump.cpp

```diff
@@ -90,7 +92,13 @@ class ZimDumper
     bool verbose;
 
   public:
-    ZimDumper(const char* fname, bool titleSort)
+    ZimDumper(const std::string& fname)
+      : file(fname),
+        pos(file.begin()),
+        verbose(false)
+      { }
+
+    ZimDumper(const std::string fname, bool titleSort)
```

Comment:
You create a second creator but you never use it.
And I think we don't need it anyway.

Reply:
I removed it.